app/src/main/java/com/github/houseorganizer/houseorganizer/task/TaskListAdapter.java

```diff
@@ -81,6 +96,7 @@ public void onBindViewHolder(@NonNull BiViewHolder<Button, Button> holder, int p
             final AlertDialog alertDialog
                     = new AlertDialog.Builder(v.getContext())
                     .setNeutralButton(R.string.add_subtask, null)
+                    .setPositiveButton("Assignees", null) // TODO extract string
```

Comment:
that TODO should probably be done before merging

Reply:
Thanks for catching it! Fixing it in the next commit